zulip/integrations/slack/slackdata2zulipdata.py

```diff
@@ -337,11 +336,28 @@ def main(slack_zip_file: str) -> None:
     # now for message.json
     message_json = {}
     zerver_message = []
-    # TODO map zerver_usermessage
+    zerver_usermessage = []
+
     for channel in added_channels.keys():
         zerver_message.append(channelmessage2zerver_message(slack_dir, channel,
                               added_users, added_channels))
+    # construct the usermessage object and append it to zerver_usermessage
+    usermessage_id = 1
+    for zulip_message in zerver_message:
+        for subscription in zerver_subscription:
+            if subscription['recipient'] == zulip_message['recipient']:
```

Comment:
Maybe should add a comment saying that this is a stream message?

Reply:
Ah, right, it loops over all the zerver_subscription, which includes streams and huddles etc, my mistake!

Reply:
@rht is this necessary? zerver_message itself is interpreted to be the stream message.